src/utils/create_component_file/index.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
```

Comment:
This file should not be here, wrong rebase?

Reply:
Probably because of switching between branches.
Removed.